website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -25,6 +25,7 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## December 2024
 
+- **Enhancement**: [Service principal](/docs/cloud/git/setup-azure#configure-the-entra-id-connection) is now available as an authentication type for the Azure DevOps integration. Service principals offer a level of security and reliability above service users, so talk to your teams about migrating.
```

Comment:
So I'm thinking these notes will probably go out before we begin the rollout. Do we want to mention something about how this feature might not be readily available for all customers right away, but if they do want to beta test it they should reach out to their sales rep or something. Not sure how we've worded this before but just so they don't expect that this isn't working when in actuality the feature flag is just off.

Reply:
Good call. We usually say "phased rollout" but since this is early release and not really rollout, I'll remove this until we do roll it out for real